pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -20,36 +20,43 @@ def load_earth_free_air_anomaly(
     ] = "01d",
     region: Sequence[float] | str | None = None,
     registration: Literal["gridline", "pixel", None] = None,
+    uncertainty: bool = False,
```

Comment:
Please change line 2 to:
```
Function to download the IGPP Earth free-air anomaly and uncertainty datasets 
from the GMT data server, and load as :class:`xarray.DataArray`.
```

Reply:
Included in comm ithttps://github.com/GenericMappingTools/pygmt/pull/3727/commits/5d8c6062f3ec99823543a25bc89df967c0fff7a2.